features/katas/basic_gates/basis_change.feature.md

```
@@ -1,5 +1,5 @@
-# Feature: Quantum Katas BasicGates Task 1.2 BasisChange
-  Task 1.2 BasisChange: |0⟩ を |+⟩ に、|1⟩ を |-⟩ に変える
+# Feature: Quantum Katas BasicGates 課題 1.2 BasisChange
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)